src/Behat/Context/UIContext.php

```diff
@@ -368,4 +368,22 @@ protected function fillFieldUnknown(NodeElement $element, $value) {
     }
   }
 
+  /**
+   * Check selector is disabled or not.
+   *
+   * @Then I check the input with :label label is disabled
+   */
+  public function inputIsDisabled($label) {
+    $session = $this->getSession();
+    $xpath = "//label[contains(text(),'" . $label . "')]/following-sibling::input";
+    $element = $session->getPage()->find(
+      'xpath',
+      $xpath
+    );
+    $disabled = $element->getAttribute('disabled');
```

Comment:
If the element does not exists this will produce a fatal error, as $element will be null. Here the code should check that $element is an instance of `Behat\Mink\Element\NodeElement`.